Cargo.toml

```diff
@@ -55,6 +55,9 @@ portpicker = { version = "0.1.1", optional = true }
 num-bigint-dig = "0.8.1"
 tracing = { version = "0.1.37" }
 
+[target.'cfg(not(target_arch = "wasm32-unknown-unknown"))'.dependencies]
+whoami = "0.5"
+
```

Comment:
Ideally, this dependency should be in `sspi-ffi` only and `custom_set_hostname` should be called with the result.

Reply:
Done